CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## 1.17.3-dev
+* Enhancement - Add support for containers to inhereit ENI private DNS hostnames for "awsvpc" tasks [#1278](https://github.com/aws/amazon-ecs-agent/pull/1278)
```

Comment:
This should be ` awsvpc` ?

Reply:
apparently it was for the 1.17.0 changelog 🤔
okay sure will make this change, although i did tell @richardpen id put quotes